### ISSUE TITLE
Reconfigure SMART Alert settings: Cannot read in config file, erroraction_config.cmd

### ISSUE THREAD

**Issue by c72578** [deajan/smartmontools-win#16]
When starting "Reconfigure SMART Alert settings" I get the following error message:
"Cannot read in config file [C:\Program Files\smartmontools for Windows\bin\erroraction_config.cmd].

This happens every time.
There is no difference if "Reconfigure SMART Alert settings" is started from the program menu entry, from the command line or as admin.
It was also tried to save and use "erroraction_config.cmd" in a different location, e.g. "e:\erroraction_config.cmd" with full permissions - no effect.

This occurs with the following versions:
smartmontools-win-6.5-1.exe
smartmontools-win-6.5-2-dev.exe

OS: Windows 7 SP1, 64 bit

![smartmontools-win_cannot_read_in_config_file](https://cloud.githubusercontent.com/assets/371551/26678609/964d6e02-46d1-11e7-90b7-245c8139c431.png)

**Comment by deajan** [comment]
I cannot reproduce this bug unless erroraction_config.exe is run without administrative privileges.
Does the erroraction_config.cmd file exist in the described path ?
Also, what does it contain please ?

**Comment by c72578** [comment]
Yes, the erroraction_config.cmd exists in the described path. Otherwise the message "Wrong configuration file path provided." appears immediately.
The content of the erroraction_config.cmd is the default content, which is installed:

:: erroraction_config.cmd file for smartmontools for Windows v6.5+

:: Valid values are: yes, no
set MAIL_ALERT=yes


set SOURCE_MAIL=
set DESTINATION_MAIL=
set SMTP_SERVER=
set SMTP_PORT=25
set SMTP_USER=

:: SMTP Password must be encoded in base64 (example via https://www.base64encode.org)
set SMTP_PASSWORD=

:: Security may be: none, ssl or tls
set SECURITY=none

:: Valid values are: yes, no
set LOCAL_ALERT=no

:: You may change this to a customized warning message or leave it to the default
set WARNING_MESSAGE=WARNUNG: Eine Ihrer Festplatten hat SMART-Fehler. Bitte schreiben Sie diese Nachricht auf und kontaktieren Sie Ihren technischen Support so schnell wie möglich.

:: Valid values are: yes, no
set COMPRESS_LOGS=yes

**Comment by deajan** [comment]
Okay, getting a bit short of ideas. Any antivirus / antimalware software that could block access to ```.cmd``` files maybe ?
Does this behavior happen on multiple machines ?
And well... double checked that you run ```erroraction_config.exe``` with UAC elevated ?

**Comment by c72578** [comment]
I could find the reason for the problem in the meantime.
It is caused by a special character in the German message. The message is set automatically during installation. When I change "möglich" to "moeglich" inside the file erroraction_config.cmd, then there is no error.

The erroraction_config.cmd file with the German message is attached. The charset of the file seems to be ANSI. You should be able to reproduce the issue with it.

OS: Windows 7 SP1, 64 bit, German

[erroraction_config.zip](https://github.com/deajan/smartmontools-win/files/1059189/erroraction_config.zip)

**Comment by deajan** [comment]
Thanks for your help, I'll investigate :)
